docs/jwst/background_step/description.rst

```diff
@@ -87,7 +87,12 @@ For Wide-Field Slitless Spectroscopy expsoures (NIS_WFSS and NRC_WFSS),
 a background reference image is subtracted from the target exposure.
 Before being subtracted, the background reference image is scaled to match the
 signal level of the WFSS image within background (source-free) regions of the
-image. 
+image. The scaling factor is determined based on the variance-weighted mean
```

Comment:
... based on the inverse variance weighted mean of the ratio image between the science data and the reference background, i.e., factor = ...
This factor is equivalent to solving for the scaling constant applied to the reference background that gives the maximum likelihood of matching the science data.